### PR TITLE
Update seed notifications

### PR DESCRIPTION
**Pull request by locano**
Just adding some changes to seed notifications with this update the notifications will show they respective activity when user makes clicks on it.

### CODE REVIEW

File: GudBox/.idea/gradle.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
       <GradleProjectSettings>
         <option name="distributionType" value="LOCAL" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleHome" value="C:\Program Files\Android\Android Studio\gradle\gradle-2.14.1" />
```
Comment (cancinos) [inline GudBox/.idea/gradle.xml]: Please, undo this change. Gradle home isn't D: disk. @locano

File: GudBox/.idea/misc.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@
     <ConfirmationsSetting value="0" id="Add" />
     <ConfirmationsSetting value="0" id="Remove" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (cancinos) [inline GudBox/.idea/misc.xml]: Don't push this change to origin. We won't downgrade jdk to version 7. @locano

**Comment by cancinos** [comment]
I'll ask you a few things before accepting your PR. First, review the commentaries that I made about your commits and secondly, could you please add more inner documentation to your classes? I mean, we need them to be as documented as possible. Comments explaining what it does, should help @locano .

File: GudBox/.idea/misc.xml (hunk 2 above)
Comment (locano) [inline GudBox/.idea/misc.xml]: i will check my JDK version because my android is taking the 1_7 but i also have the 1_8 installed.
